stdlib/bytes.gr

```diff
@@ -203,7 +202,7 @@ provide let slice = (start: Number, length: Number, bytes: Bytes) => {
   let length = coerceNumberToWasmI32(length)
   if (start + length > size) {
     throw Exception.InvalidArgument(
-      "The given index and length do not specify a valid range of bytes"
+      "The given index and length do not specify a valid range of bytes",
```

Comment:
I don't think we should add the trailing comma here. It makes it seem like the `Exception` can take multiple values when it can't.